code/game/objects/items/RCD.dm

```diff
@@ -663,8 +663,8 @@ RLD
 	icon_state = "rld"
 	lefthand_file = 'icons/mob/inhands/equipment/tools_lefthand.dmi'
 	righthand_file = 'icons/mob/inhands/equipment/tools_righthand.dmi'
-	matter = 500
-	max_matter = 500
+	matter = 200
```

Comment:
????

Reply:
You didn't copy and paste this correctly.

The RLD is overpowered and needs nerfing for it to be viable to be used in-game, it probably should be nerfed much harder than this.

Reply:
You're doing something completely unrelated to the map changes... in a map change PR. 
That's not how it works.